core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -47,7 +47,7 @@ THE SOFTWARE.
              data-cannot-enable="${%This plugin cannot be enabled}" 
              data-cannot-disable="${%This plugin cannot be disabled}" 
              data-cannot-uninstall="${%This plugin cannot be uninstalled}" 
-             data-disabled-dependencies="${%It has one or more disabled dependencies}" 
+             data-disabled-dependencies="${%It has one or more missing dependencies}"
```

Comment:
Are you sure it won't also be use when the plugin is disabled? I mean, plugins can still be wrongly disabled from the filesystem of `JENKINS_HOME`, so we should keep the `disabled` and introduce the `missing` when the plugin is simply not installed. wdyt?

Reply:
If a dependency is disabled, it's missing, would `unavailable` work better?

Reply:
Perhaps just _unsatisfied_?